.travis.yml

```diff
@@ -31,12 +31,15 @@ install:
   - env GO111MODULE=on go mod vendor
 script:
   - export GO111MODULE=on
-  - wget -q --waitretry=1 --retry-connrefused -T 30 -O - $ELASTICSEARCH_URL
+  - wget -q --waitretry=5 --tries=50 --retry-connrefused -T 600 -O - http://127.0.0.1:9200
```

Comment:
Aha! It looks like the `--tries=50` is the key to getting this to work on travis! Nice!

Reply:
Yeah, our issue was that wget was reaching 20 tries in 20sec, so it never waited 30 sec. I bumped a few parameters, 10min should never happen and might be overkill but we could always lower it a bit.